<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSingleLight</class>
 <widget class="QWidget" name="FormSingleLight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="3" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="onOffButton">
       <property name="text">
        <string>On</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ambilight</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QLabel" name="label_5">
     <property name="toolTip">
      <string>Brightness</string>
     </property>
     <property name="toolTipDuration">
      <number>3</number>
     </property>
     <property name="text">
      <string>Bri %</string>
     </property>
    </widget>
   </item>
   <item row="14" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QDial" name="hueDial">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>true</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Hue %</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="3">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="8" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Saturation %</string>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <widget class="QSlider" name="briSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="5">
    <widget class="QDial" name="satDial">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="5">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="3" colspan="3">
    <widget class="QWidget" name="groups" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Groups:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="groupsLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>satDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>briSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
